test/unit/events.spec.ts

```diff
@@ -135,6 +137,65 @@ describe("Events", () => {
     });
   });
 
+  describe("events can be stopped almost immediately", () => {
+    let timer: SinonFakeTimers;
+    beforeEach(() => {
+      timer = sinon.useFakeTimers();
+      flickingInfo = createFlicking(horizontal.full);
+    });
+    afterEach(() => {
+      timer.restore();
+    })
+
+    it("holdStart can be stopped almost immediately", () => {
+      // Given
+      let holdCount = 0;
+      const flicking = flickingInfo.instance;
+      flicking.on(EVENTS.HOLD_START, e => {
+        expect(flicking.isPlaying()).to.be.false;
+        holdCount += 1;
+        e.stop();
+      });
+
+      // When
+      simulate(flickingInfo.element, { deltaX: -70, duration: 30 });
+      timer.tick(50);
+
+      simulate(flickingInfo.element, { deltaX: -70, duration: 30 });
+      timer.tick(500);
+
+      // Then
+      expect(holdCount).equals(2);
+    });
+
+    const testingEvents = [
+      EVENTS.MOVE_START,
+      EVENTS.MOVE,
+    ]
+
+    testingEvents.forEach(event => {
+      it(`${event} can be stopped almost immediately`, () => {
+        // Given
+        let eventCount = 0;
+        const flicking = flickingInfo.instance;
+        flicking.on(event, e => {
+          e.stop();
+          eventCount += 1;
```

Comment:
Is result of previous code different from result of current code when `EVENTS.MOVE_START`?

Reply:
In `holdStart` testing code, there's more `expect` test for whether it's animating.
That test is useless for `moveStart` and `move`, as it's explicitly animating.
But, looks like I can merge both tests I think.

Reply:
@WoodNeck I mean... that previous code is not holdStart(which is above), it is not fixed code. I think this previous code will be woking fine although it's not fixed.

Reply:
That is,
it's not like something directly related to this commit but it's more like testing Axes is working correctly, which is like checking whether using correct version of Axes or so.